### DISCUSSION TITLE
Zoom Group Practical 4

### DISCUSSION THREAD

**Discussion by ISchwandner** [IBAHCM/RPiR#460] [Old]
Another discussion to share the links to your repositories for practical 4

**Comment by ISchwandner** [comment]
Hey everyone,

Here is a link to my repository:
https://github.com/IBAHCM/ISchwandnerBCI
If you could please run
(1) devtools::install_github("IBAHCM/ISchwandnerBCI") 
to check if you can install it properly 
and maybe also check if the data is all there
(2)
bci_2010
library(tibble)
bci_taxa
bci_quadrats

and generally if I am missing any important files , I would be super grateful. Will of course return the favour!

Best, Imo

**Reply by maryadejoh** [reply]
Hi Imo,
Please check if you've given me access, because it failed to install on my end.

Regards,
Mary

**Reply by ISchwandner** [reply]
Hi Mary,

Thanks for the feedback! Where you able to access my repository on GitHub at all? I checked and I gave access to the whole RPiR 21 course, so that shouldn't be the issue.

Where there any error messages when you tried to install and if so could you please share them? 

Many thanks!!

**Reply by maryadejoh** [reply]
Hi Imo,
This is what i get when I run: devtools::install_github("IBAHCM/ISchwandnerBCI")
Error: Failed to install 'unknown package' from GitHub:
  HTTP error 404.
  Not Found

  Did you spell the repo owner (`IBAHCM`) and repo name (`ISchwandnerBCI`) correctly?
  - If spelling is correct, check that you have the required permissions to access the repo.
  Let someone else try running to check if it's from my end

**Comment by maryadejoh** [comment]
Hello everyone. Please see the link to my repo and package: https://github.com/IBAHCM/maryadejohBCI.git. You can either clone my repository or use devtools::install_github("IBAHCM/maryadejohBCI".
Kindly let me know if you can run and view the data embedded.
Any feedback is appreciated

Cheers,
Mary

**Comment by rachael-clark** [comment]
Hi everyone, hoping it's okay to join the zoom group for this series? 

Here is a link to my repo: https://github.com/IBAHCM/RachaelClarkBCI.git

Was wondering if you would mind checking over my package?

**Reply by ISchwandner** [reply]
Hey Rachael,

Of course you're very welcome. Will check it now and raise an issue for you!

Best, Imo
